_pages/law/memo-guidelines.md

```diff
@@ -102,11 +102,11 @@ Use common sense. Start with the documents that reach the most people and create
 
 ### Is there someone at my agency responsible for plain language?
 
-Your agency will designate a senior official to be responsible for implementing your plain-language program. You may also have one or two other contacts in addition to this senior official. We will post all these names on the Plain Language website (www.plainlanguage.gov). [This list is currently not on the web.]
+Your agency will designate a senior official to be responsible for implementing your plain language program. You may also have one or two other contacts in addition to this senior official. We will post all these names on <plainlanguage.gov>. [This list is currently not on the web.]
 
 ### Where can I get help in rewriting my documents in plain language?
 
-Start with your agency plain language official. The Plain Language website at www.plainlanguage.gov offers more detailed guidance, good examples, related reference documents, links to other plain language sites, and a list of people willing to give you advice and assistance. If you still need help, e-mail a message to info@plainlanguage.gov. [This email no longer works. Use the "contact us" link on this website to ask for help.]
+Start with your agency plain language official. The Plain Language website at www.plainlanguage.gov offers more detailed guidance, good examples, related reference documents, links to other plain language sites, and a list of people willing to give you advice and assistance. If you still need help, email a message to <mailto:info@plainlanguage.gov>. [This email no longer works. Use the "contact us" link on this website to ask for help.]
```

Comment:
Can we use `mailto:{{ site.email }}` here instead of `mailto:info@plainlanguage.gov`?